travis-ymls/xerces-c.travis.yml

```diff
@@ -0,0 +1,86 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : xerces-c
+# Source Repo         : https://github.com/apache/xerces-c
+# Travis Job Link     : https://travis-ci.com/github/rajesh-ibm-power/xerces-c
+# Created travis.yml  : Yes
```

Comment:
Should this be "Yes"? I see a travis.yml at https://github.com/apache/xerces-c/blob/master/.travis.yml, is this not adapted from there?

Reply:
I have updated Created travis.yml = No